walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -21,7 +21,8 @@
     // TODO(issue/15): Add failure test case
 
     private static final String TEST_JOB_COLLECTION = "Jobs";
-    private static final int BATCH_SIZE = 10;
+    // should delete all the documents that the tests create
+    private static final int BATCH_SIZE = 20;
```

Comment:
As the logic of deleteCollection is that we keep deleting if count of delete >= batch size. How would large batch size ensure that we delete all documents created by test while a lower batch size does not?

Reply:
It seems like the issue might be with the database rather than something on our side. I've reverted these changes.